### DISCUSSION TITLE
PowerDNS *XFR issue after upgrading from v.4.3 to v.4.7

### DISCUSSION THREAD

**Discussion by clusterx** [poweradmin/poweradmin#509] [General]
PowerAdmin 2.1.9
Debian Buster/PHP 7.4
PowerDNS 4.3.2

I know this issue is not directly related to poweradmin, but I guess some of you have encountered a similar problem during an upgrade.
I use an external PowerDNS Authoritative Server without Dnsdist & Recursor, which updates the Linode secondary NS service via API.
Currently installed is pdns_server 4.3.2 & pdnsutil 4.3.2, after an upgrade to 4.7.4 AXFR/IXFR doesn't work anymore.
Zone changes are initiated but no longer executed (no more acknowledge or finished logs) when changing a zone.
Is it possible that TSIG is required with version 4.7.4 to allow *XFR the Linode secondary service ... or what could be the reason?

**Comment by clusterx** [comment]
maybe I need to upgrade (re-install) poweradmin afterwards, will try v3.3.0 w/ PHP7.4 which should work w/ pdns v.4.7.4

some more details:
May 14 11:22:11 ns1 pdns_server[13720]: Received spurious notify answer for 'testdomain.com' from xxx.xxx.xxx.xx:53

**Comment by edmondas** [comment]
Closing this issue as it's unrelated to PowerAdmin. The issue is already resolved upstream at https://github.com/PowerDNS/pdns/issues/12853
